lambdas/GetSnippetFromS3/lambda_function.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+from boto3.s3.transfer import ClientError
+import os
+import json
+
+s3 = boto3.client('s3', 'us-west-2')
+client = boto3.client('lambda')
+
+def lambda_handler(event, context):
+    key = event['pathParameters']['user_id'] + '/' + event['pathParameters']['snippet_id']
+    bucket = os.environ['BucketName']
```

Comment:
Please add a check to ensure the environment variable exists, and log an appropriate error if it is missing. Like so:
```python
    bucket = os.environ['BucketName']
    if(bucket == None):
        print 'Must specify "BucketName" env var!'
        raise error
```